### PR TITLE
Trev8939/sketch editor

### PR DESCRIPTION
**Pull request by TADraeseke**
adds a new sample showing use of the sketch editor API.

@shellygill we should decide where to place and/or what to do with the 'map-sketching' sample.

https://devtopia.esri.com/runtime/common-samples/issues/511

### CODE REVIEW

File: java/sketch-editor/src/main/java/com/esri/arcgisruntime/sample/mapsketchingapi/MainActivity.java [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+    mMapView.setMap(map);
+
+    mGraphicsOverlay = new GraphicsOverlay();
+    mMapView.getGraphicsOverlays().add(mGraphicsOverlay);
+
+    // create a new sketch editor and add it to the map view
+    mSketchEditor = new SketchEditor();
+    mMapView.setSketchEditor(mSketchEditor);
+
+    // inflate buttons
```
Comment (shellygill) [inline java/sketch-editor/src/main/java/com/esri/arcgisruntime/sample/mapsketchingapi/MainActivity.java]: This code is not inflating buttons, it's getting buttons from the layout.

File: java/sketch-editor/src/main/java/com/esri/arcgisruntime/sample/mapsketchingapi/MainActivity.java [hunk 2]
```diff
@@ -0,0 +1,276 @@
...
+   */
+  private void createModePoint() {
+    resetButtons();
+    mPointButton.setSelected(true);
+    mSketchEditor.start(SketchCreationMode.POINT);
+  }
+
+  /**
+   * When the multipoint button is clicked, reset other buttons, show the multipoint button as selected, and start
+   * point drawing mode.
```
Comment (shellygill) [inline java/sketch-editor/src/main/java/com/esri/arcgisruntime/sample/mapsketchingapi/MainActivity.java]: start multipoint drawing mode.

File: java/sketch-editor/src/main/java/com/esri/arcgisruntime/sample/mapsketchingapi/MainActivity.java [hunk 3]
```diff
@@ -0,0 +1,276 @@
...
+   * When the redo button is clicked, redo the last undone event on the SketchEditor.
+   */
+  private void redo() {
+    if (mSketchEditor.canRedo()) {
+      mSketchEditor.redo();
+    }
+  }
+
+  /**
+   * When the stop button is clicked, check that sketch if valid. If so, get the geometry from the sketch, set its
```
Comment (shellygill) [inline java/sketch-editor/src/main/java/com/esri/arcgisruntime/sample/mapsketchingapi/MainActivity.java]: *is* valid

**Comment by alan-edi** [comment]
Changes made. Thanks @shellygill

**Pull request merged**